assets/section-rich-text.css

```diff
@@ -25,6 +27,11 @@
 }
 
 @media screen and (min-width: 750px) {
+  .rich-text {
+    padding-top: calc(5rem + var(--page-width-margin));
+    padding-bottom: calc(5rem + var(--page-width-margin));
+  }
```

Comment:
I see you added those in a couple of sections. Email signup I think is the other one. Was that part of the original issue you're fixing?

Reply:
Thanks for pointing this out. I think they got left in during the rebased, but Tyler removed them during the padding PR.

Reply:
There is one place for the image with text where I couldn't apply it due to specific styling to that section. So the box shadow there isn't applied through a pseudo element.